### PR TITLE
[9.0][ADD] stock_inventory_verification_request

### PR DESCRIPTION
**Pull request by LoisRForgeFlow**
Stock Inventory Verification Request
====================================

Adds the capability to request a Slot Verification when a inventory is
'Pending to Approve'. When asked from Inventory Adjustment, which have
discrepancies over the threshold for the location, a Slot Verification
Request will be created for each line that exceed the maximum discrepancy
allowed.

The Inventory Manager can then confirm the Slot verification and start to
check the involved Inventory Lines and Moves to find out some possible mistake
or problem.

Usage
=====

In order to use this module act as follow:

* From a Inventory Adjustment in state 'Pending to Approve' click in the
  button 'Request Verification'. This will create all the Slot Verification
  Request needed.
* Go to 'Inventory / Inventory Control / Slot Verification Request'
* Go to a Slot Verification Request 'Waiting Actions' and confirm it.
* You can now check the involved lines and moves to help you.
* Once you have found the problem and you have fixed it 'Mark as Solved' the
  Verification.

### CODE REVIEW

File: stock_inventory_verification_request/models/stock_inventory.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+                self.env['stock.slot.verification.request'].create({
+                    'inventory_id': self.id,
+                    'inventory_line_id': line.id,
+                    'location_id': line.location_id.id,
+                    'state': 'wait',
+                    'product_id': line.product_id.id,
+                })
+
+
+class StockInventory(models.Model):
```
Comment (lasley) [inline stock_inventory_verification_request/models/stock_inventory.py]: This model needs to be renamed - you're declaring `StockInventory` twice

Comment (LoisRForgeFlow) [reply stock_inventory_verification_request/models/stock_inventory.py]: Sure. I'll attend that now! Thanks for reviewing.

**Comment by LoisRForgeFlow** [comment]
@Darshan-serpent Can you please review?

File: stock_inventory_verification_request/models/stock_slot_verification_request.py [hunk 2]
```diff
@@ -0,0 +1,133 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from openerp import fields, models, api
```
Comment (rousseldenis) [inline stock_inventory_verification_request/models/stock_slot_verification_request.py]: api, fields, models

File: stock_inventory_verification_request/models/stock_inventory.py [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+                    'inventory_line_id': line.id,
+                    'location_id': line.location_id.id,
+                    'state': 'wait',
+                    'product_id': line.product_id.id,
+                })
+
+
+class StockInventoryLine(models.Model):
+    _inherit = 'stock.inventory.line'
+
```
Comment (rousseldenis) [inline stock_inventory_verification_request/models/stock_inventory.py]: Maybe add rec_name = 'product_id' as in  Slot Verification request, you have 'stock.inventory.line, id'

**Review by rousseldenis — CHANGES_REQUESTED** [review]
Minor comments. Tested on runbot : ok

File: stock_inventory_verification_request/models/stock_inventory.py (hunk 3 above)
Comment (LoisRForgeFlow) [reply stock_inventory_verification_request/models/stock_inventory.py]: yes, that's cleaner. I'll add.

**Review by rousseldenis — APPROVED** [review]

**Review by JordiBForgeFlow — APPROVED** [review]
:+1: Technical and functional review

**Pull request merged**